clusterloader2/testing/network/config.yaml

```diff
@@ -0,0 +1,33 @@
+{{$PROTOCOL := .PROTOCOL}}
```

Comment:
And here do:
{{$PROTOCOL := .CL2_PROTOCOL}}

same below

Reply:
yes modified.